src/WebView.android.tsx

```diff
@@ -174,6 +175,8 @@ const WebViewComponent = forwardRef<{}, AndroidWebViewProps>(({
     onHttpError={onHttpError}
     onRenderProcessGone={onRenderProcessGone}
     onMessage={onMessage}
+    onOpenWindow={onOpenWindow}
+    hasOnOpenWindowEvent={onOpenWindowProp !== undefined}
```

Comment:
Are you sure those are event handlers? I don't see any code calling `onContentSizeChange` nor `onScroll`.

Reply:
Hmm by reading the code this should be the case for `onScroll`. Maybe there are some internal mapping in react-native mappers? Sorry I'm new at creating react-native bridges.

Reply:
`onScroll` is dispatched [here](https://github.com/react-native-webview/react-native-webview/blob/c8258bc3eea6fc26a3f01b02ed85297ce891523d/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java#L1762)

It looks a little different from others because `ScrollEvent` is defined by React Native, not by react-native-webview. Same story for `onContentSizeChange`, which is dispatched [here](https://github.com/react-native-webview/react-native-webview/blob/c8258bc3eea6fc26a3f01b02ed85297ce891523d/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java#L1582)